website/snippets/_bigquery-permissions.md

```diff
@@ -0,0 +1,13 @@
+BigQuery's permission model differs from conventional databases like Snowflake and Redshift. dbt uer accounts need the following permissions to read from and create tables and <Term id="view">views</Term> in a BigQuery project:
```

Comment:
From a partner perspective, one side or the other could have an issue with this part
```suggestion
dbt user accounts need the following permissions to read from and create tables and <Term id="view">views</Term> in a BigQuery project:
```